packages/database/src/migrations/20210810040153-FQSVisualsInLESMIS-modifies-data.js

```diff
@@ -0,0 +1,446 @@
+'use strict';
+
+import { insertObject, generateId, findSingleRecord, findSingleRecordBySql } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const FQS1_DATA_MAP = [
+  {
+    dataElement: 'lesmis_fqs1_stdbuilding',
+    label:
+      'All school buildings (100%) have a semi-permanent standard building / classroom or above',
+  },
+  { dataElement: 'lesmis_fqs1_office', label: 'One (1) office or teacher room per school' },
+  {
+    dataElement: 'lesmis_fqs1_cabinet',
+    label: 'One (1) cabinet or document file cabinet per school',
+  },
+  {
+    dataElement: 'lesmis_fqs1_water',
+    label: 'School has water for the use the whole year through',
```

Comment:
Assume we want to keep the funky grammar @enunan?

Reply:
"School has water for use the whole year"

Reply:
There are a few examples - should we correct each of them rather than going with what they had?

Reply:
Sorry yes on my phone I could see a whole list but on here I can only see a small snapshot
Yes please correct all to be plain english.